components/ai/code/ai-bbe/write-robust-api-powered-ai-applications.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Write robust API-powered AI applications'
-description: "Take on the unpredictable world of distributed systems with Ballerina's built-in language and standard library features. Writing robust API-powered AI applications is now a breeze for every developer, no matter the challenges!"
+description: "Take on the unpredictable world of distributed systems with Ballerina's built-in language and library features. Writing robust API-powered AI applications is now a breeze for every developer, no matter the challenges!"
```

Comment:
Here in the original text `built-in language and standard library` means : langlib and standard lib in previous terminology. Now those two are combined to one thing named Ballerina library. So it is incorrect to say `built-in language ...` in new text.

Reply:
Changed to 'with the power of the Ballerina features` as it is not required to highlight the library features only but all features in general.